### DISCUSSION TITLE
Display a section of tags as pills in the index page?

### DISCUSSION THREAD

**Discussion by aravindhsampath** [squidfunk/mkdocs-material#5519] [Q&A]
I'm trying to use mkdocs with Material theme as primarily a blog(using the blog plugin). I would like my index page to display a section of all tags contained in my posts such that if they are clicked they would show a page with a list of blog posts that contain that tag.
The current tags implementation as far as I understand, only adds a format of [tag pill followed by list of articles that have the tag],[next tag pill followed by list of articles].

Is there a way to simply print the list of tags as hyperlinks on the index page? something like this...

![image](https://github.com/squidfunk/mkdocs-material/assets/1080908/060c6111-2def-4c60-9584-acbb7580beae)

**Comment by squidfunk** [comment]
The [built-in tags plugin](https://squidfunk.github.io/mkdocs-material/setup/setting-up-tags/#built-in-tags-plugin) supports [generating a tags index page](https://squidfunk.github.io/mkdocs-material/setup/setting-up-tags/#adding-a-tags-index). Is that what you mean? It doesn't generate secondary pages per tag, though – all pages are listed below the tag on that index page.

**Reply by aravindhsampath** [reply]
No. I did see that part of documentation. But, what I am looking for is to create a list of tags, but _not_ generate the pages list below it.
Basically trying to replicate what you see on the right column of https://aravindh.net
Is it possible?

![14AE61EC-9728-4030-BE5C-4D525C93CAF1](https://github.com/squidfunk/mkdocs-material/assets/1080908/ad3eab7f-52d4-40c2-accc-cacf188e1f8f)

**Reply by squidfunk** [reply]
Currently not, and it's not planned.

**Reply by squidfunk** [reply]
However, you may extend the theme and implement the functionality yourself. It's certainly possible, but you would need to generate pages for each of the tags. That'll be tricky, but definitely possible.

**Comment by StefanoCecere** [comment]
> However, you may extend the theme and implement the functionality yourself. It's certainly possible, but you would need to generate pages for each of the tags. That'll be tricky, but definitely possible.

but hiw can i generate a tag index just for a few tags? (i don't want them all, maybe just "tag1, tag2" .. is it possible?

ah ok, found: https://squidfunk.github.io/mkdocs-material/plugins/tags/#config.tags_extra_files
